com.civicrm.hrjobroles/js/src/job-roles/controllers/add-new-role.html

```diff
@@ -225,23 +236,22 @@
                 <tr ng-repeat="(key, funder) in jobroles.editData['new_role_id']['funders']">
                   <td>
                     <!--show funder in non-edit mode-->
-                    <span ng-show="!tableform.$visible">{{funder.funder_id.sort_name}}</span>
```

Comment:
Why were we using `sort_name` and now `title`? What is the difference between two?

Reply:
The option values label were represented as [title](https://github.com/compucorp/civihr/blob/master/com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js#L535). This formed the reason for change as contact name were previously displayed using the `sort_name` column data from civicrm_contact table.